File: 351003/Suchok/publisher/src/main/java/by/liza/app/dto/response/ArticleResponseTo.java
package by.liza.app.dto.response;

import lombok.Data;
import java.io.Serializable;
import java.time.LocalDateTime;
import java.util.List;

@Data
public class ArticleResponseTo implements Serializable {
    private Long id;
    private Long writerId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
    private List<Long> markIds;
}

File: 351002/Yemyalyanenka/Task 330/discussionApi/Models/DTOs/ArticleResponseTo.cs
namespace RestApiTask.Models.DTOs
{
    public class ArticleResponseTo
    {
        public long Id { get; set; }
        public long WriterId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
    }
}
